docs/hopsworksai/azure/restrictive_permissions.md

```diff
@@ -175,3 +181,28 @@ For this replace the permissions given in the [getting started instructions](./g
     }
 ]
 ```
+
+### Upgrades
+
+If you do not intend to upgrade your cluster to newer versions of Hopsworks, then you can remove the permissions required for upgrade from your custom role that you have created [here](../getting_started.md#step-21-creating-a-restrictive-role-for-accessing-storage). For this replace the permissions in the custom role by the following permissions:
```

Comment:
```suggestion
If you do not intend to upgrade your cluster to newer versions of Hopsworks, then you can remove the permissions required for upgrade from the custom role that you have created [here](../getting_started.md#step-21-creating-a-restrictive-role-for-accessing-storage). For this replace the permissions in the custom role by the following permissions:
```

Reply:
Also I think it is better to say "remove these lines" than replace by this, because in the future we will add some other permission somewhere else and we will forget to update this part of the document.